src/dbos-runtime/runtime.ts

```diff
@@ -93,4 +94,4 @@ export class DBOSRuntime {
     }
     await this.dbosExec?.destroy();
   }
-}
\ No newline at end of file
+}
```

Comment:
Wasn't that change made in a previous PR? https://github.com/dbos-inc/dbos-ts/pull/352/files